### ISSUE TITLE
Manufactured Item scope and reusability (FHIR 4.6 vs 5.0)

### ISSUE THREAD

**Issue by rlindstrm** [hl7-eu/unicom-ig#29]
The concept of Manufactured Item is quite simple for tablets and other solid dose forms. For liquids, manufactured item is more complicated to grasp. Let's say we have two manufactured items of the same product: **5ml vial** & **2.5ml vial**. It's exactly the same content, just the size of the vial is different.

The size of the Manufactured Item (5ml or 2.5ml) is given in PackagedProductDefinition.containedItem. It is in the same attribute as the number of Manufactured Items for tablets, so it's a little confusing, but nearly ok ([see this issue](https://github.com/hl7-eu/unicom-ig/issues/11)).
So, knowing that the size of the vial is not in the Manufactured Item definition, we would think these manufactured items are identical for FHIR and could share the resource. Now, the plot thickens.

**In FHIR 4.6**, which is the basis of EMA implementations, Manufactured Item referenced Ingredients. An Ingredient includes strengths. While the concentration strength for these items would be the same (let's say 1mg/1ml), the presentation strength would be different: **5mg/1 vial** and **2.5mg/1 vial**. This means that these two manufactured items have to be separate resources, because they would have different presentation strength.

**In FHIR 5.0 and 4.3 (R4B)** the direction of referencing has been turned around ([this issue](https://github.com/hl7-eu/unicom-ig/issues/4)) and Ingredient references Manufactured Item. Now, these two manufactured items are indeed entirely identical.

- Dose form: the same;
- Unit of presentation: the same;
- Ingredient/strength - referenced from outside;
- Size of the vial - defined in PackagedProductDefinition.

While it could be convenient for implementers, and it definately increases the reusability of ManufacturedItemDefinition between products, I think it sort of questions the point of having a ManufacturedItem resource at all.

**Comment by rlindstrm** [comment]
(Concluding some additional information from other discussions)
In registers it's tempting to reuse manufactured items across medicinal products (for example, Yasminelle and Yasminelle28, where the effective pills are exactly the same, but one medicinal product includes 7 placebo pills).

However, it is not always straightforward to determine if the manufactured item between products is exactly the same (there is no identifier for that), and as different medicinal products may have marketing authorisation processes in different timelines, there might be a need to change products at different times.

And most importantly:
In SPOR PMS manufactured items are planned to be shared only within one medicinal product, but not between different medicinal products. 

(We should really separate discussions and issues.)

**Comment by Censacrof** [comment]
To me it looks like in the end we won't be able to reuse any of the IDMP enities. It would be useful to do so, but as you are saying we would need a way of determining if two entities are identical without the use of IDMP identifiers, this might unfeasible in most cases or very hard to accomplish at best. Maybe it would be wise to not reuse entities at all.

**Comment by costateixeira** [comment]
We can proceed without forcing or assuming any reuse. We just consider that
the SE products have for now different entities than EE products etc.

That doesn't block the discussion but unblocks us from moving forward and
pushes the question to where it should be answered.

**Comment by elinmaymerry** [comment]
We are implementing MPD at the Norwegian NCA and our aim is to implement reuse of the manufactured item within one Medicinal Product (MP), but not across different Medicinal Product. 

Reuse is achieved more easily on all solid dose forms, but for other dose forms we have still not clearly defined an algoritm that can identify where to reuse and where not, even within the Medicinal Product.

However, the example given above on different amount in single dose surringe, its not a valid example for our MPD. The reason is that two different size surringe may have the same concentration strenght, but because they have different presentation strength, they are not defined under the same Medicinal Product. The general practise for Marketing Authorisation is that the different size of surringe have different MT. and hence they should be different MP. If they are not, some older MT, for example Fragmin, are still operation under same MT, we are splitting them, so that it will be consistently manged and presented in the MPD along with same type of products, for example Klexane.

So our conclusion is that products with different presentation strength are not in the same MP.

But do all products have presentation strength in the Ingredient defining the Manufactured Item? 
Our conclusion so far is that presentation strength is not needed for many products with specified dose forms, for example eyedrops and semi solids (cream, gel). Hence it opens up for a wider reuse of the Manufactued Item.
The presentation strength is not useful for clinical practice, and should be left out. Only using the concentration strength, Manufactued Item can be reused across different sizes of cream tubes, f.eks Apolat Actavis, can define one Manufactued Item across cream tubed with 15g, 50g and 100g. That is only possible if the presentation strength is left out of the ingredient definition for the manufactued Item.
Same for eyedrops (multiuse) defined with concentration strength, for example 0,5 mg/ml. The Contained Item contained in the Package will then hold the amount 4ml.

**Comment by rlindstrm** [comment]
Hi @elinmaymerry! Nice to have you here. :)

Thank you for the analysis. You have a good point about concentration and presentation strength. For now, I think in cases where the Marketing Authorisation applicant wants to see the presentation strength, we are forced to have it, but maybe there are cases where we could avoid it (I should check the EMA IG about it).

I would be really happy if you looked at our other issues too (for example terminologies and mappings) and let us know if you maybe already have solutions to some of the FHIR things we're doing here. :)

**Comment by Censacrof** [comment]
Hi @elinmaymerry, thanks for the good insights. I think some time ago we discussed the use of Presentation vs. Concentration Strength with Robert. If I remember correctly, he told me that it is possible to convert from Concentration to Presentation if you have information about molecular masses for the substances. Is this a possible approach?

**Comment by rlindstrm** [comment]
I think Robert explained calculation between (salt) strength and reference strength, this is were molecular masses play a role.

Concentration and presentation strength can be calculable from eachother when you know the size of the unit of presentation. For example: 10mg/vial is presentation strength. Knowing that the size of vial is 2ml, you could easily get the concentration strength from that. Quite often, presentation strength is given as 10mg/2ml, where the size of the unit is implied in the denominator. 
In regulatory data, you don't have a lot of freedom to choose what you leave in or out. If the marketing authorisation application says 10mg/2ml, you have to go with it, even if you find it redundant or complicated.

**Comment by rlindstrm** [comment]
Briefly covered in UNICOM IG Known Issues section.
